e2e-tests/src/NPG/wallet.npg.onboarding.integration.test.js

```diff
@@ -1,99 +1,51 @@
-import { fillCardDataForm, retrieveValidRedirectUrl, getOutcome, waitUntilUrlContains, clickPaypalButton, checkAndClickPaypalFirstPsps, fillPaypalAuth, cleanWalletOnboarded, getWalletId, getWalletAlreadyOnboarded } from './helper';
+import { getOutcomeUrlForTest, registerOutcomeInterceptor, registerPageOutcomeTracker } from '../utils/outcomeUrlInterceptor';
+import { fillCardDataForm, retrieveValidRedirectUrl, getOutcome, waitUntilUrlContains, clickPaypalButton, checkAndClickPaypalFirstPsps, fillPaypalAuthAndPay, cleanWalletOnboarded, getWalletId, getWalletAlreadyOnboarded, fillPaypalAuthAndCancel } from './helper';
 
 describe('Credit Card Wallet: onboarding with NPG', () => {
   const WALLET_HOST = String(process.env.WALLET_HOST);
   const PAYMENT_METHOD_ID = String(process.env.CREDIT_CARD_PAYMENT_METHOD_ID);
-  const VALID_CARD_DATA = {
-    number: '5127390031101597',
-    expirationDate: '1025',
-    ccv: '015',
-    holderName: "TEST TEST",
-  };
   const ONBOARDED_VALID_CARD_DATA = {
     number: '5255000260000014',
     expirationDate: '1230',
     ccv: '123',
     holderName: "TEST TEST",
   };
   /**
-   * Increase default test timeout (60000ms)
+   * Increase default test timeout (2 minutes)
    * to support entire payment flow
    */
   const timeout = 120_000;
   jest.setTimeout(timeout);
-  jest.retryTimes(3);
+  jest.retryTimes(1);
```

Comment:
Only 1 retry ?

Reply:
Yes, I think that one retry is enough to recover from some transient error during test execution.
If there is a systematic error there is no need to retry tests more times also because tests are executed in azure pipelines where they could be retried each time we want.
Potentially tests have to be retried once fixed some error in the environment (that could be a PSP misconfiguration, a downstream dependency error etc): those kind of errors will not beneficiate from a test retry in the environment, not after a manual intervention to fix what is broken, so no need to retry the same test multiple times.
This way we could also make overall test execution faster allowing detecting errors earlier